platform/android/MapLibreAndroid/src/vulkan/java/org/maplibre/android/maps/renderer/textureview/TextureViewMapRenderer.java

```diff
@@ -0,0 +1,131 @@
+package org.maplibre.android.maps.renderer.textureview;
+
+import android.content.Context;
+import android.view.Surface;
+import android.view.TextureView;
+import android.view.View;
+
+import androidx.annotation.NonNull;
+
+import org.maplibre.android.maps.renderer.MapRenderer;
+
+/**
+ * The {@link TextureViewMapRenderer} encapsulates the GL thread and
+ * {@link TextureView} specifics to render the map.
+ *
+ * @see MapRenderer
+ */
+public class TextureViewMapRenderer extends MapRenderer {
```

Comment:
This class seems identical to the GL version.

Reply:
Removed